scripts/prepare_sector_network.py

```diff
@@ -171,13 +171,13 @@ def H2_liquid_fossil_conversions(n, costs):
         bus1=spatial.oil.nodes,
         bus2=spatial.co2.nodes,
         carrier="Fischer-Tropsch",
-        efficiency=costs.at["Fischer-Tropsch", "efficiency"],
+        efficiency=1 / costs.at["Fischer-Tropsch", "electricity-input"],
+        efficiency2=costs.at["Fischer-Tropsch", "hydrogen-input"]
+        / costs.at["Fischer-Tropsch", "electricity-input"],
         capital_cost=costs.at["Fischer-Tropsch", "fixed"]
-        * costs.at[
-            "Fischer-Tropsch", "efficiency"
-        ],  # Use efficiency to convert from EUR/MW_FT/a to EUR/MW_H2/a
-        efficiency2=-costs.at["oil", "CO2 intensity"]
-        * costs.at["Fischer-Tropsch", "efficiency"],
+        / costs.at["Fischer-Tropsch", "electricity-input"],

```

Comment:
Also, I think capital_cost should be multiplication of `fixed` with `efficiency` as previously defined.

Reply:
Agree, see comment below